src/main/resources/translator.properties

```diff
@@ -279,6 +279,7 @@ hudson.model.StringParameterDefinition.name = name
 hudson.model.StringParameterDefinition.description = description
 hudson.model.StringParameterDefinition.description.type = com.adq.jenkins.xmljobtodsl.dsl.strategies.custom.DSLMandatoryStringStrategy
 hudson.model.StringParameterDefinition.defaultValue = defaultValue
+hudson.model.StringParameterDefinition.defaultValue.type = com.adq.jenkins.xmljobtodsl.dsl.strategies.custom.DSLMandatoryStringStrategy
```

Comment:
Seems like you originally had the idea to make a new class but moved away from that?